user/tlstunnel/files/tmpfiles.conf

```diff
@@ -0,0 +1,3 @@
+# Create tlstunnel data directory
+
+d /var/lib/tlstunnel 0750 _tlstunnel _tlstunnel -
```

Comment:
the tmpfiles should also create `/etc/tlstunnel`

Reply:
as a convenience for the user to easily add a configuration file? okay.